internal/kafka/test/integration/kafka/kafka_test.go

```diff
@@ -41,9 +42,10 @@ func TestMain(m *testing.M) {
 	defer ocmServer.Close()
 
 	helper, _, teardown = ktest.NewKafkaHelper(t, ocmServer)
+	defer func() { os.Exit(exitCode) }()
```

Comment:
Shouldn't this be moved above the defer call to `ocmServer.Close()` ?

The order on which deferred calls are run is LIFO and I understand the Exit method call should be performed at the end.

Reply:
Thanks. Good spot. I didn't see the other deferral. Fixed.